packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -7,20 +7,6 @@ import * as api from "@fluidframework/protocol-definitions";
 import { HostStoragePolicy } from "@fluidframework/odsp-driver-definitions";
 import { ISnapshotContents } from "./odspPublicUtils";
 
-/** https://portal.microsofticm.com/imp/v3/incidents/details/308931013/home
- * The commits property was removed from protocol-definitions but in order to support back compat, we will
- * temporarily add back in this local structure in order to upload the snapshot to support rolling back to 0.58.
- * Notice this entire interface will be removed once the backward compatibility is not required anymore.
-*/
-export interface ISnapshotTreeEx {
```

Comment:
@NicholasCouri, FYI, this was introduced in https://github.com/microsoft/FluidFramework/pull/10414
It was a pain to find any good info (as two ICM tickets do not contain good info, but git history points to this PR and Teams discussion - would be great for ICM tickets to have summary of such discussions).

I believe it's safe to remove, I see no reason for us to create extra nodes that nobody cares about.

Reply:
@vladsud  - it looks good to me. Yes, I will take it as a follow-up to have the discussions added to the ICM tickets.  

Out of curiosity, when you go about removing code, don't you look at the PRs/changes that added them? There was also additional info on the teams thread linked to the PR. 

This were the repro steps:
a) create document in previously deployed (later) DF version [20220517009](https://office.visualstudio.com/OC/_build/results?buildId=13433145&view=results)

b) open the document in currently deployed (earlier) DF version [20220513008](https://office.visualstudio.com/OC/_build/results?buildId=13383396&view=results)

Reply:
I look at history only when needed :) In most cases the intention of the code is clear.